CosmosToRedis/CosmosToRedis.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using StackExchange.Redis;
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+using Container = Microsoft.Azure.Cosmos.Container;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+
+    public static class CosmosToRedis
+    {
+        public const string localhostSetting = "redisLocalhost";
+        private static readonly IDatabase cache = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(localhostSetting)).GetDatabase();
+
+        public const string Endpoint = "Endpoint";
+
+        [FunctionName("CosmosToRedis")]
+        public static void Run([CosmosDBTrigger(
+        databaseName: "databaseName",
+        containerName: "containerName",
+        Connection = "Endpoint",
+        LeaseContainerName = "leases")]IReadOnlyList<ListData> input, ILogger log)
+        {
+            if (input == null || input.Count <= 0) return;
+
+            Console.WriteLine(input);
+
+            foreach (var value2 in input)
+            {
+                RedisValue[] redisValues = Array.ConvertAll(value2.value.ToArray(), item => (RedisValue)item);
+                foreach (var value in redisValues)
+                {
+                    cache.ListRightPush("listTest", value);
```

Comment:
I do not think this name should stay `listTest` I think this should be a variable and comments should explain to the customer that this is the name of the list they can update for their use case.